src/main/kotlin/g0901_1000/s0925_long_pressed_name/Solution.kt

```diff
@@ -0,0 +1,38 @@
+package g0901_1000.s0925_long_pressed_name
+
+// #Easy #String #Two_Pointers #2023_04_17_Time_138_ms_(76.19%)_Space_33.4_MB_(100.00%)
+
+class Solution {
+    fun isLongPressedName(name: String, typed: String): Boolean {
+        var i = 0
+        var j = 0
+        var prev = '$'
+        if (typed.length < name.length) {
```

Comment:
Please add more tests for task

https://sonarcloud.io/component_measures?metric=new_coverage&selected=javadev_LeetCode-in-Kotlin%3Asrc%2Fmain%2Fkotlin%2Fg0901_1000%2Fs0925_long_pressed_name%2FSolution.kt&view=list&pullRequest=327&id=javadev_LeetCode-in-Kotlin